### ISSUE TITLE
Tooltip: Links in form validation tooltip in Light theme are barely visible

### ISSUE THREAD

**Issue by dprokop** [grafana/grafana#49169]
Not sure if it's me and my colour blindness, but I find links in tooltips in Light theme to be not-so-much-accessible:

<img width="989" alt="image" src="https://user-images.githubusercontent.com/2376619/169056374-7de1b2bc-fcbc-4510-ace0-2a9e56a31872.png">

At first sight, I thought that the Tooltip itself has a semi-transparent background, and the "Click..." link is a glitch from the layer below.

**Comment by adithyaakrishna** [comment]
@dprokop I would like to work on this 👀

**Comment by StephenKing** [comment]
For me, it's even more unreadable in panel links (8.5.2)
<img width="118" alt="Screenshot 2022-05-20 at 13 45 49" src="https://user-images.githubusercontent.com/277561/169521792-27be4282-f11a-404a-aca7-919888ef1f48.png">

**Comment by joshhunt** [comment]
@StephenKing That should be fixed in https://github.com/grafana/grafana/pull/48748, which will be backported to the next point release. You can check main if you would like to confirm.

**Comment by StephenKing** [comment]
Thanks @joshhunt, missed that one

**Comment by joshhunt** [comment]
Thanks for the offer @adithyaakrishna - this one required a bit of back and forth to find the right place (and colors) to use, so I'll tackle this myself.
